ts/src/pro/ascendex.ts

```diff
@@ -912,22 +923,30 @@ export default class ascendex extends ascendexRest {
         //     { m: 'sub', id: '1647515701', ch: 'depth:BTC/USDT', code: 0 }
         //
         const channel = this.safeString (message, 'ch', '');
-        if (channel.indexOf ('depth-realtime') > -1) {
-            this.handleOrderBookSubscription (client, message);
+        const id = this.safeString (message, 'id');
+        const subscriptionsById = this.indexBy (client.subscriptions, 'id');
+        const subscription = this.safeValue (subscriptionsById, id);
+        if (subscription !== undefined) {
+            const method = this.safeValue (subscription, 'method');
+            if (method !== undefined) {
+                return method.call (this, client, message, subscription);
+            }
+            // clean up
+            if (id in client.subscriptions) {
+                delete client.subscriptions[id];
+            }
         }
         return message;
     }
 
-    handleOrderBookSubscription (client: Client, message) {
-        const channel = this.safeString (message, 'ch');
-        const parts = channel.split (':');
-        const marketId = parts[1];
-        const symbol = this.safeSymbol (marketId);
+    handleOrderBookSubscription (client: Client, message, subscription) {
```

Comment:
same here,  `handleOrderBookSubscription`  were implemented on all other exchanges with additional argument in signature, and I've changed ascendex's implementation for this method too, to be similar as all other exchanges have it.